SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -805,20 +816,72 @@ UObject* USpatialReceiver::GetTargetObjectFromChannelAndClass(USpatialActorChann
 void USpatialReceiver::OnReserveEntityIdResponse(Worker_ReserveEntityIdResponseOp& Op)
 {
 	UE_LOG(LogSpatialReceiver, Log, TEXT("Received reserve entity Id: request id: %d, entity id: %lld"), Op.request_id, Op.entity_id);
+
 	if (USpatialActorChannel* Channel = PopPendingActorRequest(Op.request_id))
 	{
 		Channel->OnReserveEntityIdResponse(Op);
 	}
 }
 
-void USpatialReceiver::OnCreateEntityIdResponse(Worker_CreateEntityResponseOp& Op)
+void USpatialReceiver::OnReserveEntityIdsResponse(Worker_ReserveEntityIdsResponseOp& Op)
```

Comment:
Do we intend to modify the Worker_ReserveEntityIdsResponseOp or can it be const?

Reply:
Can probably be const

Reply:
https://improbableio.atlassian.net/browse/UNR-657